packages/@ourworldindata/grapher/src/mapCharts/ChoroplethGlobe.tsx

```diff
@@ -464,7 +464,11 @@ export class ChoroplethGlobe extends React.Component<{
         }
 
         // rotate to the selected country on mobile
-        if (!this.manager.isMapSelectionEnabled) {
+        if (
+            !this.manager.isMapSelectionEnabled &&
+            // don't switch to 3D if the user is zoomed in, otherwise they can get stuck in 3D mode
+            window?.visualViewport?.scale === 1
+        ) {
```

Comment:
No need to do this here since we're already on the globe and clicking on a country only rotates to it

Reply:
Oh yeah, you're right. I was thinking that this would help prevent people getting stuck in this case:
1. Enter globe mode
2. Scroll down page
3. Pinch zoom
4. Scroll back up
5. Click on globe

But I don't think it makes a difference. If they're zoomed in enough, they're stuck regardless 😬 

Should be pretty rare at least!